host_core/lib/host_core/claims/manager.ex

```diff
@@ -82,7 +83,8 @@ defmodule HostCore.Claims.Manager do
           Enum.join(claims.tags, ",")
         end,
       version: claims.version,
-      sub: claims.public_key
+      sub: claims.public_key,
+      contract_id: Map.get(claims, :contract_id)
```

Comment:
If this is an actor, we shouldn't add the contract ID here (or at least it should be "n/a") since making it null can cause deserialization errors down the line